### ISSUE TITLE
Prepare for release

### ISSUE THREAD

**Issue by dafnevk** [NLeSC/SalientDetector-python#43]
* Regenerate docuementation
* Make changelog
* Make github release
* Change status on software.esciencecenter.nl

**Comment by dafnevk** [comment]
* Release on Pypi (http://peterdowns.com/posts/first-time-with-pypi.html)

**Comment by elboyran** [comment]
Are we also doing [semantic versioning](http://semver.org/)?
I'm not sure git does it by default.

**Comment by dafnevk** [comment]
Yes, we need to carefully choose the next version number. But I think we can go for 1.0.0, since this is the first stable release and the API should be stable from now on.

**Comment by elboyran** [comment]
Will you also change the status and links (currently the source code points only at the Python code) @ software.esciencecenter.nl?

**Comment by dafnevk** [comment]
I actually don't think it's possible to link to two github repositories from the NLeSC site.
But maybe we should update the homepage (github pages) so that it also clearly links to the matlab repository and describes both.

**Comment by elboyran** [comment]
Isn't it possible to simply have 2 buttons (or 2 hyper links) one for the MATLAB source code, the other for the Python on the NLeSc site or the template is quite fixed as number of elements and links?
